### ISSUE TITLE
Remove unused fields in preset spreadsheet

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#776]
I'll flag a list of fields when I have the time to do this and run it by you before removing @jimmyrocks.

**Comment by jimmyrocks** [comment]
I think all of those fields are used. Maybe we could come up with some better explanations for the field uses.

**Comment by chadlawlis** [comment]
I don't believe `optionalTags`, `description`, or `howToMap` are being used. Are all four symbol fields being used? `npmapSymbol`, `hfcSymbol`, `iconSource`, and `makiIcon`?
